models/rgcn_semantic_encoder.py

```diff
@@ -19,14 +19,15 @@ def __init__(self, args):
 
         self.n_graph_attn_composition_layers = args.n_graph_attn_composition_layers
         self.output_size = self.transformer.config.hidden_size
+        self.graph_dim = args.graph_dim
 
         if self.use_semantic_graph:
-            self.emb_proj = nn.Linear(self.transformer.config.hidden_size, args.graph_dim)
-
+            self.emb_proj = nn.Linear(self.transformer.config.hidden_size, self.graph_dim)
+            
```

Comment:
Sorry, perhaps I wasn't being clear enough. Do you mind removing the leading spaces here? The rest looks great!

Reply:
Sorry that I still didn't get your point. May I ask which line should be changed?

Reply:
If you look at the diff, your change introduced some leading whitespace on the new L26 (the old L25). It's ok if you leave it as is if it's too troublesome to change -- I'm happy to fix that after merging this PR. Thanks for your contribution again!

Reply:
Oh, I got it. Would remove them and commit. Thanks for your reminder.